src/encoded/static/components/item-pages/HealthView.js

```diff
@@ -235,18 +239,15 @@ export default class HealthView extends React.PureComponent {
         if (microMetaFrom && microMetaFrom.indexOf('#') > -1) {
             [ microMetaVersionUsed ] = microMetaFrom.split('#').splice(-1);
         }
-        //extend context to include vitessce version
-        const { dependencies } = installedPackageLockJson;
-        let vitessceVersion;
-        if (dependencies && dependencies.vitessce) {
-            vitessceVersion = dependencies.vitessce.version;
-        }
+        //extend context to include higlass & vitessce version
+        const { dependencies: { 'higlass': { version: higlassVersion } = {}, 'vitessce': { version: vitessceVersion } } } = installedPackageLockJson;
```

Comment:
Nice cleanup. I think we might do something like this for CGAP's health page, too, when I do this update.